apps/server/server.cpp

```diff
@@ -33,12 +33,17 @@
 #include "aditof/aditof.h"
 #include "aditof/sensor_enumerator_factory.h"
 #include "aditof/sensor_enumerator_interface.h"
+#ifdef PROTOBUF_DEFINED
```

Comment:
The server runs on target, never on PC. When you disable protobuf and glog, you did it for PC.
Also, the way the server is written, cannot function without protobuf.